components/host-dashboard/AcceptReject.js

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Mutation } from 'react-apollo';
+import gql from 'graphql-tag';
+
+import StyledButton from '../StyledButton';
+import AppRejectionReasonModal from './AppRejectionReasonModal';
+
+const ACollectiveMutation = gql`
```

Comment:
Explicit names are always better

```suggestion
const ApproveCollectiveMutation = gql`
```

Reply:
Yeah, It was proof of concept. I am going to make it better